### DISCUSSION TITLE
Video files and AI - HELP

### DISCUSSION THREAD

**Discussion by alyr0007** [community/community#140228] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

I am new to all of this, I thought I would join to talk to some smart people !!!!

I have a challenge.

 I am a sports coach and I’m interested in the ability to have a sequence of play in a game (say rugby union) which is captured in video and convert it into animation. 
 
The animation I’m looking for is where all the players become dots and move around the field.  Currently we have the ability to do this manually which takes time. 

Does anyone know a piece of software or AI platform that can help?

look forward to hearing from you clever people !!!

‘cheers
Al

**Comment by Xelliese** [comment]
Automating the conversion of video to animation, especially for sports analysis, sounds like it could save a lot of time and offer deeper insights. You might want to look into software solutions that specialize in motion capture and analysis; however, many of these might require some manual setup initially. For capturing the original video sequences before conversion, you could use a tool like [https://www.screencapture.com/](https://www.screencapture.com/), which can help you easily record any live action directly from your computer screen.

**Comment by alyr0007** [comment]
Many thanks, I have extensive background in using motion capture and
analysis software, I find it as you say time consuming. I was curious that
with AI if there was a solution that would enable me to convert the clips
to animation and get solutions by using the athletes to ask it questions.

I was hoping someone out there might be able to solve the problem without a
massive cost.

Thanks for replying
